### PR TITLE
Added routes for fetching and submitting reviews

### PR DESCRIPTION
**Pull request by SwankSwashbucklers**
Note: Requires updated API endpoint: https://github.com/tablelist/api/pull/601

### CODE REVIEW

File: src/review/service.js [hunk 1]
```diff
@@ -5,5 +5,19 @@ angular
...
 		var ReviewService = Service.extend(Review);
 
+		ReviewService.prototype.read = function read(options) {
+			if (!options) throw new Error('options is required');
+			if (!options.id) throw new Error('options.id is required');
+
+			return Review.get(options).$promise;
+		};
+
+		ReviewService.prototype.update = function update(options) {
```
Comment (alexjohnson505) [inline src/review/service.js]: Updating a review isn't currently supported on the API right?

**Comment by alexjohnson505** [comment]
Don't forget to increment the version! Since we have a number of different clients (web, admin, affiliates, etc) using tablelist.js, it's good to keep track.

https://github.com/tablelist/tablelist.js/blob/master/bower.json#L3

File: src/review/service.js (hunk 1 above)
Comment (SwankSwashbucklers) [inline src/review/service.js]: It's supported, the iOS app uses it

**Comment by officert** [comment]
looks good :+1:

**Comment by alexjohnson505** [comment]
@AndrewBarba Merge?

**Pull request merged**